plugins/Login/Controller.php

```diff
@@ -362,6 +360,9 @@ public function resetPasswordSuccess()
      */
     public static function clearSession()
     {
+        $sessionFingerprint = new Session\SessionFingerprint();
+        $sessionFingerprint->clear();
+
         $authCookieName = Config::getInstance()->General['login_cookie_name'];
```

Comment:
is this still needed considering the cookie has been deprecated and is not set anymore?

Reply:
The existing cookie will still be there (ie, it won't automatically be unset for all users once this PR is merged), so we have to explicitly remove it somewhere (since it will have the token auth in it). I thought it was safer to just keep this. I can change it if you have a better idea, though.

Reply:
I was hoping the original cookie will eventually expire?

Reply:
Yes you're right, it should just expire. I'll remove this code.